src/Services/UpdateExport.php

```diff
@@ -28,6 +29,8 @@ public function run(): Export
 
         $export->save();
 
+        event(new ExportCompleted($export));
```

Comment:
I've moved this from the app to here, seems to make more sense to dispatch the event from the package.

What do you think?

Reply:
agreed!